.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: python
-sudo: false
 cache: pip
 
 python:
-  - "3.4"
+  - "3.5"
```

Comment:
Any particular reason this change was made in this pull request?

Reply:
yes it was causing problems in the build

Reply:
Okay. Thanks!